homework/index.js

```diff
@@ -32,39 +35,59 @@
 
   const root = document.getElementById('root');
   const header = createAndAppend('header', root, { class: 'header' });
```

Comment:
Lines 36-37 should not be at the file level, but inside function `main()` so that they get executed along with `window.onload`.

Reply:
I try but the code gave an error.